sdk/appcenter/src/androidTest/java/com/microsoft/appcenter/ingestion/models/one/PartCUtilsAndroidTest.java

```diff
@@ -40,6 +40,14 @@ public void nullKey() {
         PartCUtils.addPartCFromLog(properties, log);
     }
 
+    @Test(expected = IllegalArgumentException.class)
```

Comment:
Can you also add a test with empty string value?

Reply:
Done